### PR TITLE
Docs 5031 Web Service Consumer Reference 1.5

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/web-service/web-service-consumer-configure.adoc [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+= Configure the Web Service Consumer
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: core, connector, web service consumer, WSS, service, consumer, wsdl, soap
 
 To consume a SOAP Web Service operation, you need to create a configuration that points to the service that you want to consume.
 
-The Web Service Consumer (WSC) connector has a single configuration. The most important configuration parameters are the the WSDL Location (`wdslLocation`), the Service (`service`), and the Port (`port`). The configuration will not be valid if any of them are not set.
+The Web Service Consumer Connector has a single configuration. The most important configuration parameters are the the WSDL Location (`wsdlLocation`), the Service (`service`), and the Port (`port`). The configuration will not be valid if any of these parameters are not set.
```
Comment (Bardammu) [inline modules/ROOT/pages/web-service/web-service-consumer-configure.adoc]: Remove one `the`

**Review by Bardammu — CHANGES_REQUESTED** [review]
There is just a typo

**Review by Bardammu — APPROVED** [review]
LGTM

**Review by kevintroller — APPROVED** [review]
LGTM

**Pull request merged**